models/topics.model.js

```diff
@@ -92,3 +92,17 @@ exports.updateArticleVotes = async (article_id, inc_votes) => {
 
   return result.rows[0];
 };
+exports.deleteCommentById = async (comment_id) => {
+  await checkExist("comments", "comment_id", comment_id);
```

Comment:
You don't need to check that the comment exists, psql would throw an error if it didn't